pymoose/moosemodule.cpp

```diff
@@ -3114,26 +3032,6 @@ PyMODINIT_FUNC MODINIT(_moose)
 {
     clock_t modinit_start = clock();
 
-    //PyGILState_STATE gstate;
```

Comment:
tests are standalone python scripts; they are no longer embedded in `_moose.so`. Removed this un-maintained code.